chunky/src/java/se/llbit/chunky/renderer/export/PictureExportFormat.java

```diff
@@ -50,12 +52,20 @@ default String getDescription() {
   String getExtension();
 
   /**
-   * Check if this format supports transparency (used for transparent sky).
+   * Note: It depends on the scene settings if the alpha buffer will be available on export.
    *
-   * @return True if this format supports transparency, false otherwise
+   * @return the required format for the alpha buffer or {@link AlphaBuffer.Type#UNSUPPORTED} if alpha is not supported.
+   */
+  default AlphaBuffer.Type transparencyType() {
```

Comment:
Rename to keep Java getter style:

```suggestion
  default AlphaBuffer.Type getTransparencyType() {
```